ldoc/markup.lua

```diff
@@ -275,6 +275,20 @@ local formatters =
                                                     { smart = true })
          return function(text) return parse(text) end
       end
+   end,
+   pandoc = function(format)
+     return function(text)
+       local out_filename = os.tmpname()
+       local in_file = io.popen("pandoc -f markdown -t html -o " .. out_filename .. " -", "w")
```

Comment:
Overall this PR looks good and I'd love to see it merged, but this bit of coding seems counter intuitive to me. You're writing to something called "in_file" and reading from "out_file" — but it's really tricky to realize whose perspective on in and out is being used here. I suggest using names that better represent what they are ... tmpfile, stdin or fifo or some such?

Otherwise I'll give this a test drive and make sure it works, then merge.

Reply:
Maybe `markdown_file` and `html_file` (based on the contents)?

Reply:
Sure, that sounds clear.